website/docs/r/nsxt_distributed_firewall.html.markdown

```diff
@@ -12,6 +12,12 @@ description: |-
 The Distributed Firewall allows user to segment organization virtual data center entities, such as
 virtual machines, based on virtual machine names and attributes. 
 
+~> There is a different resource
```

Comment:
In addition to "yellow" and "blue" notes, is there a "red" note/warning? :) It's so critical not to mix the two ways of using (as it can lead to data corruption), that I'd like to highlight this more.

Reply:
Changed. here is how it looks 
<img width="701" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/7258b9ba-616a-486e-b9d6-55308e0a36aa">

Reply:
Serious, as it should be? To clarify: is it only "as they will override each other" or "that using both may lead to unexpected resulting rules (data corruption)"?

Reply:
More like the second case ("that using both may lead to unexpected resulting rules (data corruption)"). Using only multiple instances of `vcd_nsxt_distributed_firewall` would cause them to override each other.

Reply:
Then I'd suggest stating that. I mean, something like this:
"One should use only one of ... as using both may lead to unexpected resulting rules (data corruption)".

Reply:
Adjusted. Or do you think we need to mention the "data corruption" explicitly?

<img width="717" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/5b20934c-59eb-4f45-90fb-92df1f9cf334">